src/components/Pool/BorrowRepayComponent.vue

```diff
@@ -74,13 +76,7 @@
       />
     </div>
 
-    <div
-      v-if="
-        actionType === 'borrow' &&
-        !this.poolsWithoutLeveradge.includes(this.pool.name)
-      "
-      class="config-box"
-    >
+    <div v-if="actionType === 'borrow' && false" class="config-box">
```

Comment:
This will be never shown

Reply:
correct, that was the purpose of the change to hide it :)